src/main/java/MainClass.java

```diff
@@ -1,10 +1,13 @@
 public class MainClass {
     public static void main(String[] args) {
-        XmlMessage aaa = new XmlMessage("bla bla");
-        System.out.println(aaa.getName());
-        aaa.prepareMessage("Alex","Fasii");
-        System.out.println("body= "+aaa.getBody());
-        MessageProcessor a = new MessageProcessor();
-        a.printXml(aaa);
+        XmlMessage xmlMessage = new XmlMessage("Super message");
+        System.out.println(xmlMessage.getName());
+        System.out.println("body= " + xmlMessage.getBody());
+        xmlMessage.prepareMessage();
+        xmlMessage.prepareMessage("Something new");
+        xmlMessage.prepareMessage("Json", "Facility");
```

Comment:
could you check if this line doesn't give you a compilation error as you deleted the *prepareMessage(String,String)** method?